android/app/src/main/AndroidManifest.xml

```diff
@@ -38,6 +38,21 @@
                 <action android:name="android.intent.action.MAIN"/>
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
+            <intent-filter>
+                <action android:name="android.intent.action.SEND" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <data android:mimeType="image/*" />
```

Comment:
We should share any file, not image only.

Reply:
@donpui Sorry i pushed AndroidManifest.xml by accident MainActivity.kt

Reply:
All good, free to ignore. I was testing only error messages